src/masquerade/providers/open_sgid.py

```diff
@@ -21,6 +23,18 @@
 
 connection = psycopg2.connect(database=DATABASE, user=AGRC, password=AGRC, host=HOST)
 
+directions = ['north', 'south', 'east', 'west']
+normalize_direction_infos = []
+for direction in directions:
+    #: build a list of all of the different ways to type the direction
+    #: e.g. (no, no., nor, nor., ...)
+    permiatations = []
+    for index in range(len(direction)):
+        permiatation = direction[0:index + 1]
```

Comment:
```suggestion
        permutation = direction[0:index + 1]
```

lines 31-36: 'permiatation' should be 'permutation'